cli/tests/jsunit_tests.rs

```diff
@@ -0,0 +1,38 @@
+mod integration;
```

Comment:
This is effectively a duplicate of `cli/tests/js_unit_tests.rs`, it should be removed